Description: Build with GTK3 instead of GTK2
 - wxwidgets is built with gtk3, so we have to build with gtk3
 - add the LDFLAGS for gtk+-3.0, otherwise, the build would fail with:
 .
 /usr/bin/ld: /tmp/cc6adrRa.ltrans115.ltrans.o: undefined reference to symbol 'gtk_css_provider_load_from_path'
 /usr/bin/ld: /lib/x86_64-linux-gnu/libgtk-3.so.0: error adding symbols: DSO missing from command line
 .
 "::gtk_css_provider_load_from_path" is called in
   FreeFileSync/Source/application.cpp
   FreeFileSync/Source/RealTimeSync/application.cpp
 .
Author: B Stack
Source: Original research
Date: 2020-08-31
Forwarded: not-needed

--- a/FreeFileSync/Source/Makefile
+++ b/FreeFileSync/Source/Makefile
@@ -20,9 +20,10 @@
 CXXFLAGS  += `$(PKG_CONFIG) --cflags libssh2`
 LDFLAGS += `$(PKG_CONFIG) --libs   libssh2`
 
-CXXFLAGS  += `$(PKG_CONFIG) --cflags gtk+-2.0`
+CXXFLAGS  += `$(PKG_CONFIG) --cflags gtk+-3.0`
+LDFLAGS += `$(PKG_CONFIG) --libs gtk+-3.0`
 #treat as system headers so that warnings are hidden:
-CXXFLAGS  += -isystem/usr/include/gtk-2.0
+CXXFLAGS  += -isystem/usr/include/gtk-3.0
 
 #support for SELinux (optional)
 SELINUX_EXISTING=$(shell $(PKG_CONFIG) --exists libselinux && echo YES)
--- a/FreeFileSync/Source/RealTimeSync/Makefile
+++ b/FreeFileSync/Source/RealTimeSync/Makefile
@@ -11,9 +11,10 @@
 LDFLAGS += `$(WX_CONFIG) --libs std, aui, richtext --debug=no` -pthread
 
 #Gtk - support "no button border"
-CXXFLAGS  += `$(PKG_CONFIG) --cflags gtk+-2.0`
+CXXFLAGS  += `$(PKG_CONFIG) --cflags gtk+-3.0`
+LDFLAGS += `$(PKG_CONFIG) --libs gtk+-3.0`
 #treat as system headers so that warnings are hidden:
-CXXFLAGS  += -isystem/usr/include/gtk-2.0
+CXXFLAGS  += -isystem/usr/include/gtk-3.0
 
 cppFiles=
 cppFiles+=application.cpp
